src/tests/profiler/gc/gc.cs

```diff
@@ -6,53 +6,67 @@
 
 namespace Profiler.Tests
 {
-    class GCBasicTests //: ProfilerTest
+    class AllocObject
     {
-        static readonly Guid GcBasicEventsProfilerGuid = new Guid("A040B953-EDE7-42D9-9077-AA69BB2BE024");
+        public static readonly int ArraySize = 100;
+        int[] m_array = null;
 
-        public static void DoWork() {
-            int k=0;
-            while(k<3) {
-                Console.WriteLine("{0}: Restarting run {1}",Thread.CurrentThread.Name,k);
-                int[] largeArray = new int[1000000];
-                for (int i=0;i<=100;i++){
-                    int[] saveArray = largeArray;
-                    largeArray = new int[largeArray.Length + 100000];
-                    saveArray = null;
-                    //Console.WriteLine("{0} at size {1}",Thread.CurrentThread.Name,largeArray.Length.ToString());
-                }
-                k++;
-           }
-        }
-
-        public static int RunTest(String[] args) {
-            long Threads = 1;
-
-            if(args.Length > 2)
+        public AllocObject(bool poh)
+        {
+            if (poh)
             {
-                Console.WriteLine("usage: LargeObjectAlloc runtest <number of threads>");
-                return 1;
+                m_array = GC.AllocateArray<int>(ArraySize, true);
             }
-            else if(args.Length == 2)
+            else
             {
-                Threads = Int64.Parse(args[1]);
+                m_array = new int[ArraySize];
             }
+        }
+    }
 
-            Console.WriteLine("LargeObjectAlloc started with {0} threads. Control-C to exit",
-                Threads.ToString());
+    class GCBasicTests
+    {
+        static readonly Guid GcBasicEventsProfilerGuid = new Guid("BCD8186F-1EEC-47E9-AFA7-396F879382C3");
```

Comment:
Using 'GCBasic' here is confusing, this isn't the GCBasic profiler GUID. 

Similarly the class name was confusing when there is another GCBasicTests class in the gcbasic.cs

Reply:
Thanks, I just missed those when doing the rename